content/blog/2023-03-22-etc-and-bitcoin-have-the-fifthening-and-the-halving-in-2024/index.md

```diff
@@ -0,0 +1,131 @@
+
+**Era 2:** 4.00 ETC, from block 5,000,000, which happened on December 11, 2017
+
+**Era 3:** 3.20 ETC, from block 10,000,000 which happened on March 17, 2020
+
+**Era 4:** 2.56 ETC, from block 15,000,000 which happened on April 25, 2022
+
+Because ETC is now on block 17,088,524 as of this writing, it is paying 2.56 ETC per block as we are in era 4.
+
+In the next era, which will be era 5, the block reward will be reduced to 2.048 ETC.
+
+## When Will ETC’s Next Fifthening Happen?
+
+To estimate when the next ETC fifthening will happen we need to know how many blocks are left to the next fifthening. Then, we need to multiply that number of blocks by the average block time of the network in seconds. This will give us the number of seconds left to the next fifthening. If we transform that amount into days, then we will be able to add the result to the current date and determine when the next fifthening will happen.
+
+**Following the above logic we will do the following calculation:**
+
+Current ETC block at the time of this writing: 17,088,524
+
+Next fifthening block number: 20,000,000
+
+Blocks left to the next fifthening: 20,000,000 - 17,088,524 = 2,911,476
+
+Seconds left to the next fifthening: 2,911,476 x 13.1525 = 38,293,188
+
+Days left to the next fifthening: 38,293,188 / 60 / 60 / 24 = 443.2081
+
+Current date at the time of this writing: March 8, 2023
+
+**Current date + 443.2081 days = May 25, 2024**
+
+## What is the Halving in Bitcoin?
+
+![The BTC halving.](./2.png)
+
+Bitcoin’s monetary policy has a rule that every 210,000 blocks it will reduce the mining reward by 50%. This is called the halving because it is a 1/2 cut in the issuance of the coin.
+
+The mining reward is the amount of BTC that is issued to pay for miners to produce blocks for the network.
+
+In BTC, a block is currently being produced every 9.2660 minutes on average.
+
+The historial schedule of payments per block to miners by the Bitcoin network has been the following:
+
+**Era 1:** 50.00 BTC, from block 0, which happened on January 03, 2009
+
+**Era 2:** 25.00 BTC, from block 210,000, which happened on November 28, 2012
+
+**Era 3:** 12.50 BTC, from block 420,000 which happened on July 09, 2016
+
+**Era 4:** 6.25 BTC, from block 630,000 which happened on May 11, 2020
+
+Because Bitcoin is now on block 779,961 as of this writing, it is paying 6.25 BTC per block as we are in era 4.
+
+In the next era, which will be era 5, the block reward will be reduced to 3.125 BTC.
+
+## When Will Bitcoin’s Next Halving Happen?
+
+To estimate when the next BTC halving will happen we need to know how many blocks are left to the next halving. Then, we need to multiply that number of blocks by the average block time of the network in minutes. This will give us the number of minutes left to the next halving. If we transform that amount into days, then we will be able to add the result to the current date and determine when the next halving will happen.
```

Comment:
Line 87: Correct preposition usage
```suggestion
To estimate when the next BTC halving will happen we need to know how many blocks are left until the next halving. Then, we need to multiply that number of blocks by the average block time of the network in minutes. This will give us the number of minutes left until the next halving. If we transform that amount into days, then we will be able to add the result to the current date and determine when the next halving will happen.
```